.gitignore

```diff
@@ -3,4 +3,36 @@
 *.suo
 /Debug
 /GeneratedFiles
-build
\ No newline at end of file
+build

```

Comment:
lol I think you'll find your "git life" much easier if you opt for shadow builds, e.g.: 
```
$ cd qamqp
$ mkdir build && cd build
$ qmake ..
$ make -j5
```

All tests are rpathed to the libraries built, so you can then just do a ```make check``` to run all tests